scripts/pip-compile.sh

```diff
@@ -8,5 +8,5 @@ if ! python -c "import sys; assert sys.version_info.major == $major and sys.vers
   exit 1
 fi
 
-cd ./requirements && $(MAKE) *.txt
+cd ./requirements && make *.txt && cd ..
```

Comment:
Can use `pushd`/`popd` to navigate to/from the requirements dir. Has some pleasantries: [Navigating with popd](https://opensource.com/article/19/8/navigating-bash-shell-pushd-popd#navigating-with-popd)